packages/manager/apps/web/client/app/core/translations/Messages_fr_FR.json

```diff
@@ -122,7 +122,7 @@
   "common_ipv4": "IPv4",
   "common_ipv6": "IPv6",
   "common_bitFormat": "bits",
-  "common_service_expired": "Le service {{t0}} n'a pas été renouvelé. Cela est peut être dû à une décision de votre part ou un retard de paiement de votre côté.",
+  "common_service_expired": "Ce service {{t0}} est suspendu. Rendez-vous dans la section <a href=\"{{t1}}\">Mes Services</a> pour le réactiver.",
```

Comment:
Could this sentence split in txo parts? This way we could avoid to have HTML markup into the translations